intermediate_source/memory_format_tutorial.py

```diff
@@ -84,29 +118,51 @@
 print(y.stride()) # Ouputs: (3072, 1, 96, 3)
 
 ######################################################################
-# Pointwise operators preserves memory format
+# Pointwise operators preserves memory format. Note: for p-wise
+# operatros with multiple inputs of different memory format,
+# contiguous memory format is the dominating memory format. This means
```

Comment:
Going to be changed in https://github.com/pytorch/pytorch/pull/37968

Reply:
Thanks. The code change does spit out a warning of output format.

I'll remove this section then.